.github/workflows/graph-db.yml

```
@@ -0,0 +1,46 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: graph-db Node.js CI
+
+on:
+  push:
+    paths:
+      - 'graph-db/**'
```

Now that we see the builds working we can add this back in

```suggestion
on:
  push:
    branches: [ master ]
    paths:
      - 'graph-db/**'
```